notebooks/MAST/HSC/HSC_TAP/HSC_TAP.ipynb

```diff
@@ -0,0 +1,472 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"title_ID\"></a>\n",
+    "# MAST Table Access Protocol Hubble Source Catalog Demo"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<br> This tutorial demonstrates how to use astroquery to access the Hubble Source Catalog via a Virtual Observatory standard Table Access Protocol (TAP) service at MAST, and work with the resultant data. \n",
+    "\n",
+    "***\n",
+    "### Table of Contents\n",
+    "\n",
+    "1. [HSC TAP Service Introduction](#HSC-TAP-Service-Introduction)\n",
+    "2. [Imports](#Imports)\n",
+    "3. [Display Tweaks](#Display-Tweaks)\n",
+    "4. [Connecting, Finding, and Displaying Table Information](#Connecting,-Finding,-and-Displaying-Table-Information)\n",
+    "    - [Connecting to a TAP Service](#Connecting-to-a-TAP-Service)\n",
+    "    - [Querying for Table Schema Information](#Querying-for-Table-Schema-Information)\n",
+    "    - [Querying for Data](#Querying-for-Data)\n",
+    "    - [Use Case: Plotting a light curve for the most variable object in a field](#Use-Case:-Plotting-a-light-curve-for-the-most-variable-object-in-a-field)\n",
+    "    - [Use Case: Create a color magnitude diagram for the Small Magellanic Cloud](#Use-Case:-Create-a-color-magnitude-diagram-for-the-Small-Magellanic-Cloud) <br><br>\n",
+    "5. [Additional Resources](#Additional-Resources)\n",
+    "6. [About This Notebook](#About-this-Notebook)\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## HSC TAP Service Introduction\n",
+    "<div style=\"text-align: right\">[(top)](#title_ID)</div>"
```

Comment:
These ``<div>`` tags seem to be breaking the links. In general mixing html in with markdown is a bit problematic anyway though, so perhaps just remove them? (here and everywhere else in the notebook)
```suggestion
    "[(top)](#title_ID)"
```

or alternatively just drop the "top" links alltogether?  Personally I don't think these are particularly helpful (and will become less so when I manage to get the table of contents on the sidebar)

Reply:
Sounds good, I added this to match some other MAST notebooks style-wise and am not attached to it.